internal/ethapi/api.go

```diff
@@ -1013,6 +1013,14 @@ func (e *revertError) ErrorData() interface{} {
 // Note, this function doesn't make and changes in the state/blockchain and is
 // useful to execute and retrieve values.
 func (s *BlockChainAPI) Call(ctx context.Context, args TransactionArgs, blockNrOrHash rpc.BlockNumberOrHash, overrides *StateOverride) (hexutil.Bytes, error) {
+	num, ok := blockNrOrHash.Number()
+	genesis := s.b.GetGenesis()
+	if ok && uint64(num) < genesis.Number().Uint64() {
+		var result hexutil.Bytes
+		err := s.b.GetSequencerRPCService().CallContext(ctx, &result, "eth_call", args, blockNrOrHash, overrides)
```

Comment:
Should this really be the sequencer RPC service? We can fallback historical calls to non-sequencer nodes.

Reply:
Right, changing this to use `HistoricalRPCService()`.